public/data/binary.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": 0.1,
+    "name": "Binary",
+    "size": 5000,
```

Comment:
All map sizes should be a multiple of 2000 to prevent the background from sticking out